packages/modules/desktop-viewer-react/src/services/Initializer.ts

```diff
@@ -107,8 +107,11 @@ export class DesktopInitializer {
           DesktopInitializer._initializing = false;
           DesktopInitializer._cancel = undefined;
         });
-    } else {
-      this._initialized = Promise.resolve();
+    } else if (this._initializing) {
```

Comment:
What would happen if _initializing is false, we don't have any condition to handle the case - 

![image](https://user-images.githubusercontent.com/86079065/182694035-1ea3e76c-6e9e-42b9-af3b-977f9188b544.png)

Reply:
good catch.